apps/fluent-tester/src/TestComponents/Theme/ThemeTest.tsx

```diff
@@ -50,7 +51,12 @@ const Panel: React.FunctionComponent = () => {
       <PrimaryButton onClick={onClick} content="Primary Button" disabled={disabled} />
       <Button onClick={onClick} content="Default Button" disabled={disabled} />
       <StealthButton onClick={onClick} content="Stealth Button" disabled={disabled} />
-      <Text testID={THEME_TESTPAGE}>This is a text element</Text>
```

Comment:
If we provide a testID below (after line 111), we can remove this Text element:

```
{
    name: 'Component Examples',
    testID: THEME_TESTPAGE,
    component: Panel,
},
```

Reply:
Added.